inquestlabs.py

```diff
@@ -195,9 +195,9 @@ def __API (self, api, data=None, path=None, method="GET", raw=False):
                 attempt += 1
 
             # retries exhausted.
-            if attempt == self.retries:
+            if attempt == self.num_retries:
                 message = "exceeded %s attempts to communicate with InQuest Labs API endpoint %s."
-                message %= self.retries, endpoint
+                message %= self.num_retries, endpoint
```

Comment:
GOOD catch :-) Do we have a test case that will cover mock failure at 1 or more attempts?

Reply:
yes we do! Line 33 of test_api.py covers lines 198-201, where retries are tested